src/Database/LibSQLConnection.php

```diff
@@ -145,14 +116,4 @@ public function quote(string $value): string
     {
         return $this->escapeString($value);
     }
-
-    protected function isRunningMigrations()
```

Comment:
I don't think that package should know about some commands outside of package, add those inside package or remove them?